### ISSUE TITLE
Migrate from MongoDB to Redis

### ISSUE THREAD

**Issue by SwarnimWalavalkar** [covidarmy/backend#26]

**Comment by brainless** [comment]
Basically this ticket is about removing the following:

- [ ] 1. Code to `get` / `set` to the cache, in API endpoint handler
- [ ] 2. Remove `redis` from `npm` packages
- [ ] 3. Remove Redis related configuration if any
- [ ] 4. Remove deployment related Redis steps/config if any

**Comment by rgxai** [comment]
Yes but we are not moving forward with Redis now right? @brainless

**Comment by brainless** [comment]
I think the discussion yesterday was that MongoDB makes our life easier, and our loads can be tackled with it. So remove Redis is one less component. Please correct me if I am mistaken @rgxai

I just re-used this existing issue for this.

**Comment by rgxai** [comment]
Yes makes sense. Will rename the issue.
